cpu/cortexm_common/panic.c

```diff
@@ -16,14 +17,21 @@
  *
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ * @author      Toon Stegen <toon.stegen@altran.com>
  */
 
 #include "cpu.h"
 #include "lpm.h"
+#include "log.h"
+
+#ifdef DEVELHELP
+static void print_IPSR(void);
```

Comment:
Do we need the function prototype? Why not the function declaration here and 3 lines and a macro less?